### PR TITLE
Meta-set nodedef: Part 3

### PR DESCRIPTION
**Pull request by Thomas--S**
See #5895 for part 1.
See #5902 for part 2.

More will follow here.

This is (mainly) for all API-related changes.

### CODE REVIEW

File: src/script/common/c_content.cpp [hunk 1]
```diff
@@ -715,127 +715,176 @@ ContentFeatures read_content_features(lua_State *L, int index)
...
+}
+
+void push_content_features(lua_State *L, const ContentFeatures &f)
+{
+	std::string paramtype(ScriptApiNode::es_ContentParamType[(int)f.param_type].str);
+	std::string paramtype2(ScriptApiNode::es_ContentParamType2[(int)f.param_type_2].str);
+	std::string drawtype(ScriptApiNode::es_DrawType[(int)f.drawtype].str);
+	std::string liquid_type(ScriptApiNode::es_LiquidType[(int)f.liquid_type].str);
 	
 	/* Missing "tiles" because I don't see a usecase (at least not yet). */
```
Comment (bigfoot547) [inline src/script/common/c_content.cpp]: You should remove this line as you have added tiles.

Comment (Thomas--S) [inline src/script/common/c_content.cpp]: It is already removed in my local version, but there are still some issues with tiles in this table. I will upload an updated version as soon as it works correctly. Thanks for your notice!

**Comment by Thomas--S** [comment]
Removing the WIP tag, as it seems to be quite ready.

**Comment by Thomas--S** [comment]
If someone wonders why there is no documentation yet: I'll add documentation when meta-set nodedef is fully implemented.

**Pull request closed without merging**